src/components/PersonInfo/PersonInfo.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import cn from 'classnames';
+
+import { PersonLink } from '../PersonLink/PersonLink';
+import { Person } from '../../types';
+
+type Props = {
+  person: Person;
+  selectedPersonSlug: string;
+};
+
+const EMPTY_TABLE_CELL = '-';
```

Comment:
- this is not a cell stored inside the variable
- great job creating a constant 👍